package.json

```diff
@@ -9,13 +9,18 @@
     "test:unit": "vitest --environment jsdom --root src/"
   },
   "dependencies": {
+    "@headlessui/vue": "^1.7.4",
+    "@heroicons/vue": "^2.0.13",
+    "@ownego/polaris-vue": "^1.3.0",
     "@tailwindcss/forms": "^0.5.3",
     "@tailwindcss/line-clamp": "^0.4.2",
     "@tailwindcss/typography": "^0.5.8",
+    "core-js": "^3.8.3",
     "pinia": "^2.0.26",
     "postcss-cli": "^10.1.0",
     "postcss-nested": "^6.0.0",
     "postcss-nesting": "^10.2.0",
+    "remixicon": "^2.5.0",
```

Comment:
Why do we need this icons package?

Reply:
Separate components for Inputs, buttons, and other form elements are being created.
The mixing of styles of APIs has been removed.
Code has been formatted with "prettier"
"remixicon" has been removed and created icon as separate components